Values update (#2039)
PostHog/posthog.com: contents/handbook/company/values.md

Building a huge community around a free-for-life product is key to [PostHog's strategy](/handbook/strategy/strategy).

> We default to transparency with everything we work on. That means we make public our handbook, [our roadmap](/handbook/strategy/roadmap), [how we pay](/handbook/people/compensation) (or even [let go of](/handbook/people/offboarding)) people, [what our strategy is](/handbook/strategy/strategy), and [who we have raised money from](/handbook/strategy/investors). Internally, we go even further - providing financial information, live updates on fundraising and board slide access.

Reviewer: ```suggestion
We default to transparency with everything we work on. That means we make public our handbook, [our roadmap](/handbook/strategy/roadmap), [how we pay](/handbook/people/compensation) (or even [let go of](/handbook/people/offboarding)) people, [what our strategy is](/handbook/strategy/strategy), and [who we have raised money from](/handbook/strategy/investors). Internally, we go even further - providing financial information, live updates on fundraising, and board slide access.
```

This enables the strongest community growth possible. It causes the core team to raise the bar on their work, it provides the context needed for people to work across multiple timezones, and it enables a deep work-heavy and meeting-light culture. It creates trust.

## We haven't built our defining feature yet

We will never stop iterating.

> You learn faster, and thus help PostHog perform better by getting what you're working on into the real world. We expect you to ship new designs, features or whatever is needed for your role in tiny chunks, frequently, and often before you feel ready.

Reviewer: ```suggestion
You learn faster and help PostHog perform better by getting what you're working on into the real world. We expect you to ship new designs, features, or whatever is needed for your role in tiny chunks, frequently, and often before you feel ready.
```

## Everyone codes

> ...although this doesn't mean everyone has to be a software developer, and not everyone needs experience in this before they join. Our platform's adoption starts with developers using our open source project, thus we use GitHub to share most of our work publicly, and to build a large community of technical users.

Reviewer: ```suggestion
...although this doesn't mean everyone has to be a software developer, and not everyone needs experience in this before they join. Our platform's adoption starts with developers using our open source project so we use GitHub to share most of our work publicly and to build a large community of technical users.
```

No matter your role, being able to use the basics of GitHub helps you understand our audience. Beyond that, we'll encourage you to build your technical skill, rather than delegating more challenging tasks to others, so you become a more effective contributor.

## Step on toes

We have ambitious goals. This means people need to pick up what they know is most important at any time, not what they feel their job title means they are allowed to do.

We expect you to pick out the very most important thing you can think of, and work on that. Discard plans as you see fit.

We judge your performance based on the impact you deliver overall, no matter what your role.

## Talent compounds

Getting into PostHog is a huge challenge. Once you're here, it stays that way. We are *extremely* demanding of performance.

In return, you get to work with others producing the best work of their careers.

> We are a team, not a family. This means we have very ambitious goals, [pay top of market](/handbook/people/compensation), offer [exceptional benefits](/handbook/people/benefits), and do everything we can to provide an environment for you to do your best work. Often this means everyone, [especially managers](/handbook/company/management), getting out of your way. It's also [not ok to let your teammates fail](/handbook/company/culture/#dont-let-others-fail). We expect everyone to provide direct feedback, to help everyone perform at their best. We pay [generous severance](/handbook/people/compensation#severance) if things aren't working out.

Reviewer: ```suggestion
We are a team, not a family. This means we have very ambitious goals, [pay top of market](/handbook/people/compensation), offer [exceptional benefits](/handbook/people/benefits), and do everything we can to provide an environment for you to do your best work. Often this means everyone, [especially managers](/handbook/company/management), getting out of your way. It's also [not ok to let your teammates fail](/handbook/company/culture/#dont-let-others-fail). We expect everyone to provide direct feedback to help everyone perform at their best. We pay [generous severance](/handbook/people/compensation#severance) if things aren't working out.
```

## Bias for action

Think a change should be made? Then make it.

> Most decisions are reversible, so a culture that defaults to (slow) consensus-building is suboptimal. We'll perform better by taking 2 rapid steps forward, and 1 back, than by not progressing at all.

Reviewer: ```suggestion
Most decisions are reversible, so a culture that defaults to (slow) consensus-building is suboptimal. We'll perform better by taking two rapid steps forward, and one back, than by not progressing at all.
```
